README.md

```diff
@@ -126,6 +126,19 @@ usage: que [options] [file/to/require] ...
 
 You may need to pass que a file path to require so that it can load your app. Que will automatically load `config/environment.rb` if it exists, so you shouldn't need an argument if you're using Rails.
 
+## Usage with Active Job
+
+If you would rather integrate Que with Active Job, you can do it by setting the adapter in `config/application.rb` or in a specific environment by setting it in `config/environments/production.rb`, for example:
+```ruby
+ActiveJob::Base.queue_adapter = :que
+```
+
+Que will automatically pick up the database from Active Record, so there is no need to configure anything else.
```

Comment:
I think you mean `Active Job`, right?

Reply:
When I read [this part of the code](https://github.com/que-rb/que/blob/master/lib/que.rb#L79-L84) I thought the database connection was managed by Active Record. In fact I think you can use Que with Active Record without using Active Job at all, but I might be wrong.

Does that make sense?

Reply:
Ah. Now I get it. This says that `que` will automatically use the Active Record connection.

I think the `pick up` is confusing. Can we use something like:

> Que will automatically use the database configuration of your rails application.

Reply:
@giovannibonetti What do you think?

Reply:
Done. Anything else?